replace_words.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/python
+import sys
+import itertools
+import os
+import argparse
+
+
+class WordsModifier:
+    default_input_file = 'file1.txt'
+
+    def __init__(self, input_file, output_file, from_list_arr, to_list_arr):
+        self.input_file = input_file
+        self.output_file = output_file
+        self.from_list_arr = from_list_arr
+        self.to_list_arr = to_list_arr
+
+    def __open_file_to_read_or_write(self, file_rw, opr, text=None):
+        if opr == 'r':
+            if os.path.isfile(file_rw):
+                try:
+                    with open(file_rw, opr) as f:
+                        input_text = getattr(f, 'read')()
+                        return input_text
+                except IOError:
+                    print "Connot open input file!"
+            else:
+                print "File {0} does not exist, {1} used instead"\
+                        .format(file_rw, default_input_file)
+                input_text = open_file_to_read_or_write(default_input_file,
+                                                        opr)
+                return input_text
+
+        elif opr == 'w+':
+            try:
+                with open(file_rw, opr) as f:
+                    getattr(f, 'write')(text)
+            except IOError:
+                print "Connot open output file!"
+        else:
+            print "You use wrong operation!!"
+        return None
+
+    def replace_words(self):

```

Comment:
should return a value of the replaced string.

and there should be another function to store in a file.